multicluster/test/e2e/service_test.go

```diff
@@ -27,75 +27,108 @@ import (
 	e2euttils "antrea.io/antrea/test/e2e/utils"
 )
 
-func testServiceExport(t *testing.T, data *MCTestData) {
-	data.testServiceExport(t)
+func TestMCService(t *testing.T) {
+	data, err := setupTest(t)
```

Comment:
I don't think I see why we have to setupTest again in here in addition to what we have in `main_test.go`. Should we just move the "TestMCServiceExport" testcases there?

Reply:
Hmm.. So you mean we should trigger the test cases from main_test.go, instead of having separate TestXxx test functions in service_test.go?

Reply:
Yes, that's why I moved the service test trigger to main_test.go, which is to avoid setting up test twice

Reply:
Cool, will fix!